index.d.ts

```diff
@@ -1 +1,4 @@
-export default function<T = any>(candidates: Record<string, T>): T
+export default function<T = any>(
+  candidates: Record<string, T>,
+  defaultValue: any,
```

Comment:
Why not like that? Even default should conform to same type imo.
```suggestion
  defaultValue: T,
```

Reply:
@fredyc I don't really think it would work right
default value actually can be anything

Reply:
I don't see why it should be considering it's basically another value from those `candidates`. Besides, you are returning `T` from the whole call, so even if `defaultValue` would be something else, it would probably fail. Not sure exactly.

Reply:
Hmmm, maybe you're right
good point